scripts/install

```diff
@@ -386,22 +317,24 @@ fi
     #print_level: debug
     ynh_replace_string --match_string="print_level: debug" --replace_string="print_level: warn" --target_file="$final_path/config.yaml"
 
-cd $final_path
+#cd $final_path
 #Generate the appservice registration file by running ./mautrix-whatsapp -g.
     #You can use the -c and -r flags to change the location of the config and registration files. They default to config.yaml and registration.yaml respectively.
-$final_path/mautrix-whatsapp -g -r $synapse_config_path/$app-registration.yaml
-cd $base_directory
+$final_path/mautrix-whatsapp -g -r ../conf/registration.yaml
+#cd $base_directory
 #Add the path to the registration file (registration.yaml by default) to your synapse homeserver.yaml under app_service_config_files.
-# A list of application service config files to use
-#
-#app_service_config_files:
-#  - app_service_1.yaml
-#    - "/etc/matrix-synapse/registration.yaml"
-ynh_replace_string --match_string="#app_service_config_files:" --replace_string="app_service_config_files:" --target_file="$synapse_config_path/homeserver.yaml"
-ynh_replace_string --match_string="#  - app_service_1.yaml" --replace_string="  - '$synapse_config_path/$app-registration.yaml'" --target_file="$synapse_config_path/homeserver.yaml"
-#You will then need to restart the synapse server. Remember to restart it every time the registration file is regenerated.
+cp ../conf/registration.yaml $app_service_registration_path/$app.yaml
+
+#$as_token=;
+#$hs_token=;
+#ynh_replace_string --match_string=__AS_TOKEN__ --replace_string="$as_token" --target_file=$app_service_registration_path/$app.yaml
+#ynh_replace_string --match_string=__HS_TOKEN__ --replace_string="$hs_token" --target_file=$app_service_registration_path/$app.yaml
+if ! /opt/yunohost/matrix-$synapse_instance/update_synapse_for_appservice.sh
+then
+  ynh_die "Synapse can't restart with the appservice configuration"
+fi
```

Comment:
```suggestion
/opt/yunohost/matrix-$synapse_instance/update_synapse_for_appservice.sh \
    || ynh_die "Synapse can't restart with the appservice configuration"
```

(imho this style is slightly more semantic but that's really a super minor suggestion idk)